vp/node/views.py

```diff
@@ -228,11 +230,8 @@ def execute_node(request, node_id):
                      })
 @api_view(['GET'])
 def retrieve_data(request, node_id):
-    # TODO: currently hard coding the name of the file as workflow + node id;
-    #       will need to change the word workflow for the workflow name. Will also need to add some exceptions.
-    #
-    file_name = 'workflow-' + str(node_id)
-
+    # TODO: need to add validation probably
+    file_name = Workflow.generate_file_name(request.pyworkflow.workflow_name(), node_id)
```

Comment:
I'm not familiar with the FileStorage stuff, but do we not need `fs.open(...)` to read the file from /tmp?

Reply:
It worked fine as is. But I'll give it a try with fs, since seems to be a better way of doing it anyway.